shell/app-shell/elements/shell-stores.js

```diff
@@ -9,17 +9,17 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
 */
 
 import Xen from '../../components/xen/xen.js';
-import ArcsUtils from '../lib/arcs-utils.js';
+import ArcsUtils from '../../lib/arcs-utils.js';
 import './arc-store.js';
 import {FbStore} from './fb-data/FbStore.js';
 
 // templates
 const template = Xen.html`
-  <arc-store arc="{{arc}}" data="{{themeData}}" options="{{themeStoreOptions}}" on-change="_onShellThemeChange"></arc-store>
+  <!-- <arc-store arc="{{arc}}" data="{{themeData}}" options="{{themeStoreOptions}}" on-change="_onShellThemeChange"></arc-store>
   <arc-store arc="{{arc}}" data="{{userStoreData}}" options="{{userStoreOptions}}"></arc-store>
-  <arc-store arc="{{arc}}" data="{{usersStoreData}}" options="{{usersStoreOptions}}"></arc-store>
+  <arc-store arc="{{arc}}" data="{{usersStoreData}}" options="{{usersStoreOptions}}"></arc-store>-->
```

Comment:
What's the difference between `userStoreData` and `usersStoreData`? Not needed for this PR but naming so it's clearer could be useful

Reply:
shell-stores is deprecated; but `userStoreData` is current user data, `usersStoreData` is the list of all users.